projects/kit/components/input-inline/input-inline.template.html

```diff
@@ -24,6 +24,7 @@
     [id]="id"
     [ngModel]="value"
     [readOnly]="readOnly"
+    [style.padding-left.px]="indent"
```

Comment:
if I remember correctly padding can't be negative, it is mistake?

Reply:
fixed